rcode-reader/src/main/java/eu/rssw/pct/elements/DataRelationElement.java

```diff
@@ -0,0 +1,52 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import eu.rssw.pct.RCodeInfo;
+
+public class DataRelationElement extends AbstractElement {
+  // private int pairCount;
+  // private int flags;
+
+  private String parentBufferName;
+  private String childBufferName;
+  private String fieldPairs;
+
+  public DataRelationElement(byte[] segment, int currentPos, int textAreaOffset, ByteOrder order) {
+    // this.pairCount = ByteBuffer.wrap(segment, currentPos, Short.BYTES).order(order).getShort();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)